src/Symfony/Component/PropertyInfo/PropertyInfoExtractor.php

```diff
@@ -18,33 +18,18 @@
  */
 class PropertyInfoExtractor implements PropertyInfoExtractorInterface
 {
-    /**
-     * @var PropertyListExtractorInterface[]
-     */
     private $listExtractors;
-
-    /**
-     * @var PropertyTypeExtractorInterface[]
-     */
     private $typeExtractors;
-
-    /**
-     * @var PropertyDescriptionExtractorInterface[]
-     */
     private $descriptionExtractors;
-
-    /**
-     * @var PropertyAccessExtractorInterface[]
-     */
     private $accessExtractors;
 
     /**
-     * @param PropertyListExtractorInterface[]        $listExtractors
-     * @param PropertyTypeExtractorInterface[]        $typeExtractors
-     * @param PropertyDescriptionExtractorInterface[] $descriptionExtractors
-     * @param PropertyAccessExtractorInterface[]      $accessExtractors
+     * @param iterable|PropertyListExtractorInterface[]        $listExtractors
```

Comment:
Anyway, the RFC for introducing an `iterable` keyword has been accepted: https://wiki.php.net/rfc/iterable